routes/order.js

```diff
@@ -19,7 +19,7 @@ module.exports = function (express) {
       data.uuid = uuid_generator.generateUUID();
       data.timestamp = timestamp.makeTimestamp();
       // ending the response and console logging the response data
-      console.log(data);
+      res.send({healthy: true});
```

Comment:
This is just a demo data for showing how Unit Testing is handled. This should be switched back to actually displaying the data generated upon request